### PR TITLE
Web Inspector: align the percentages in the audit tab

### PR DESCRIPTION
**Pull request by francescorn**
#### aa1d5f4e1fe3fa3c97240ccd3a567916d391a919
<pre>
Web Inspector: align the percentages in the audit tab
<a href="https://bugs.webkit.org/show_bug.cgi?id=268207">https://bugs.webkit.org/show_bug.cgi?id=268207</a>

Reviewed by Devin Rousso.

Updating text-align:center to text-align:end for the percentages in the audit tab.

Amend changes:
* Source/WebInspectorUI/UserInterface/Views/AuditTestGroupContentView.css:
(.content-view.audit-test-group &gt; header &gt; .percentage-pass):

Combined changes:
* Source/WebInspectorUI/UserInterface/Views/AuditTestGroupContentView.css:
(.content-view.audit-test-group &gt; header &gt; .percentage-pass):

Canonical link: <a href="https://commits.webkit.org/274078@main">https://commits.webkit.org/274078@main</a>
</pre>

https://github.com/WebKit/WebKit/commit/e180273856d517b8c18763d65f6d78c5aef7aa30

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/36141 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/15088 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/38343 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/38862 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/32/builds/32492 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/17526 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/51/builds/12116 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/31186 "Passed tests") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/36696 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/47/builds/12741 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/32064 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/11172 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/42/builds/11193 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/10/builds/32245 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/40108 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/32806 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/36/builds/32612 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/37133 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/44/builds/11390 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/50/builds/9277 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/35219 "Passed tests") | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/46/builds/13114 "Built successfully") | | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/8492 "Built successfully and passed tests") | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/43/builds/11867 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/45/builds/12227 "Built successfully") | | |

### CODE REVIEW

**Comment by francescorn** [comment]
Not ready for review yet.

**Comment by francescorn** [comment]
Here is the audit tab currently in Safari 17.3 on the left and my change in the mini-browser on the right:
audit_tab_percentages_safari_on_left_and_mini_browser_on_right

**Comment by francescorn** [comment]
The above image with borders added around the percentages:
same_image_with_borders_around_percentages

**Comment by francescorn** [comment]
Ready for review.

**Comment by francescorn** [comment]
The alignment in the percentages currently works when the window is maximally sized, but not when the window is minimized.

**Comment by Ahmad-S792** [comment]
@francescorn - this needs to be reviewed by someone with `reviewer` permissions and then it needs to be added to `merge-queue` by someone with committer permissions (which I can do but not former since I am not reviewer).

**Comment by francescorn** [comment]
I saw that `safe-merge-queue` was being applied to commits with all of the EWS checks passing so I tried it.

**Comment by Ahmad-S792** [comment]
> I saw that `safe-merge-queue` was being applied to commits with all of the EWS checks passing so I tried it.

Either it will have `reviewed` (or approved by) in it or someone might have taken review by person 1-1 (in office / slack / branch or revert patch so previous review apply) and modified the commit message with `Reviewed by ABC` rather than `Reviewed by OOPS` (which we have).

**Review by dcrousso — APPROVED** [review]

**Pull request merged**
